### PR TITLE
Migrate Relay pages to bedrock behind a DEV setting

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Migrate Relay pages to bedrock behind a DEV setting.

## Significant changes and points to review

The migration work is mostly done but we need to co-ordinate launch with the Relay team. Getting this on the dev server will let the l10n team start their string migration and let the Relay team do some QA.

Most of the work in this PR has already been reviewed as previous pull requests to this branch.

- Folder structure, [sub nav](https://github.com/mozilla/bedrock/pull/13289), [favicon](https://github.com/mozilla/bedrock/pull/13194)
- [Landing page](https://github.com/mozilla/bedrock/pull/13196)
- [Premium page](https://github.com/mozilla/bedrock/pull/13196)
- [Pricing plans and geolocation logic](https://github.com/mozilla/bedrock/issues/13204) for displaying them
  - including [unit tests ](https://github.com/mozilla/bedrock/pull/13321)
- [FAQ page](https://github.com/mozilla/bedrock/pull/13118)
- Waitlist signup pages for: email, [phone, and VPN bundle](https://github.com/mozilla/bedrock/pull/13061).
- All of the above include migrated string IDs from Relay

Still outstanding:
- Stripe Plan IDs for DEV (these are just "TKTK" right now)
- functional tests
- analytics
- documentation

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/12876

## Testing

What does need to be reviewed is views.py, urls.py, and the static bundles file. Though, if you see anything else do say something.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
✅  [Integration tests are passing](https://github.com/mozilla/bedrock/actions/runs/5406043573)

**Review by craigcook — APPROVED** [review]
This all looks good, with the understanding that most parts have been pre-reviewed separately. I would feel better if a back-ender double-checked the Python bits, so a timid r+ from me.

There also seems to be some formatting error in test_helper_misc.py that should be addressed before it can merge.

**Comment by stephaniehobson** [comment]
Relay is talking about expanding the number of locales they are translated in so I'm going to put DNM on this until I can talk to someone in the L10N team about if we should delay migrating the strings.

**Review by peiying2 — COMMENTED** [review]
@stephaniehobson my review only covers the *.ftl files and nothing else. 
- It looks good, with just one "add-on" coded as brand. We can treat it as normal text.
- For each page, what is the URL? right now, they are all kind of generic: `### URL: https://www-dev.allizom.org/products/relay/`. I need the info for context when localizers or the vendor translate and test the pages.
- There are some differences on how to indicate a new version to replace obsolete strings between Relay and the mozorg pages. Relay in the past just add `xyz-2` or `xyz-3` in the string ID to indicate the newer version. Later on some use `v2` instead. This shouldn't be a concern for the migration, just want to call out. And we are not fixing the content.

File: l10n/en/brands.ftl [hunk 1]
```diff
@@ -42,6 +42,7 @@
 -brand-name-firefox-nightly = Firefox Nightly
 -brand-name-firefox-reality = Firefox Reality
 -brand-name-firefox-daylight = Firefox Daylight
+-brand-name-firefox-addons = Firefox Add-ons
```
Comment (peiying2) [inline l10n/en/brands.ftl:45]: I didn't see any Firefox Add-ons in the text, so it's safe to remove it here.

File: l10n/en/brands.ftl [hunk 2]
```diff
@@ -54,6 +55,7 @@
 -brand-name-nightly = Nightly
 -brand-name-reality = Reality
 -brand-name-daylight = Daylight
+-brand-name-addons = Add-ons
```
Comment (peiying2) [inline l10n/en/brands.ftl:58]: We can remove this as discussed in the Slack channel.

File: l10n/en/products/relay/landing.ftl [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+how-it-works-section-forward-headline = We’ll forward emails, phone calls, and texts to you
+how-it-works-section-forward-body = Rest assured, senders will never know your real email address or real number. You can even reply to texts and emails without sharing your real identity.
+
+how-it-works-section-manage-headline = Manage your masks from your { -brand-name-relay } dashboard
+how-it-works-section-manage-body = Sign in to your { -brand-name-relay } dashboard to create new masks, label existing masks, and  block calls, texts, or emails from masks that get too much spam.
+
+## REVIEWS SECTION
+
+relay-landing-brand-addons = { -brand-name-addons }
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply l10n/en/products/relay/landing.ftl:35]: I will change it back to `landing-reviews-add-ons = Add-ons` which is what relay was using.

**Pull request merged**